shared/chat/conversation/messages/coinflip/index.js

```diff
@@ -39,63 +43,99 @@ class CoinFlip extends React.Component<Props, State> {
     }, 0)
     return `${revealed} / ${total}`
   }
+  _renderStatusText = () => {
+    if (this.props.showParticipants) {
+      const participants = (
```

Comment:
🐑 can put the ismobile check here and then its just `{participants}` below

Reply:
I'm not sure how to do that. The `isMobile` check is to either next it in a `Text` block or not.